### PR TITLE
Update MCP page. Fixes #743

### LINKED ISSUE #743: Improve Using MCP in Plugins page
https://docs.spongepowered.org/stable/en/plugin/internals/mcp-setup.html

* Update the sample MCP mapping + forge version
* Use stable in the mcp mapping example (because we recommend using it)
* Maybe: Add the SpongeAPI dependency entry, so the entire config can (more or less) just be copied.
* Mention that using the mapping for a wrong version will most likely not work. (With strange error messages)

What do you think?

### PR DESCRIPTION
**Pull request by ST-DDT**
Fixes #743 

I tested the instructions while creating a dummy plugin for the entity AI docs.

### CODE REVIEW

**Comment by Spongy** [comment]
A preview for this pull request is available at https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/3acaf79/.

Here are some links to the pages that were modified:

- modified: https://cdn.rawgit.com/Spongy/SpongeDocs-PRs/3acaf79/plugin/internals/mcp-setup.html

_Since the preview frequently changes, please link to [this comment](https://github.com/SpongePowered/SpongeDocs/pull/746#issuecomment-414759246), not to the direct url to the preview._

**Comment by ST-DDT** [comment]
I'll add fixes for #585 tomorrow.

File: source/plugin/internals/mcp-setup.rst [hunk 1]
```diff
@@ -83,14 +89,26 @@ Forge Workspace
...
     minecraft {
-        forgeVersion = '2584' // TODO: Configure Forge build here
-        // TODO: Replace with your mappings version, e.g. snapshot_20180106
+        // TODO: Configure Forge build here
+        forgeVersion = '2705'
+        // TODO: Replace with your mappings version, e.g. stable_39 or snapshot_20180814
         mappings = 'YOUR_MAPPINGS_VERSION'
     }
 
+It is recommended to use the same Forge version as used by the SpongeForge build, you are going to use.
```
Comment (Inscrutable) [inline source/plugin/internals/mcp-setup.rst]: remove the comma.

File: source/plugin/internals/mcp-setup.rst [hunk 2]
```diff
@@ -83,14 +89,26 @@ Forge Workspace
...
+
+.. note::
+
+    Make sure you use a version that belongs to your Minecraft version. Using a wrong version will probably cause
```
Comment (Inscrutable) [inline source/plugin/internals/mcp-setup.rst]: belongs -> corresponds

**Review by Inscrutable — COMMENTED** [review]
Will there be more coming to this PR?

**Comment by ST-DDT** [comment]
@Inscrutable That was my original intention, but the other issue (#585) is somewhat dependent on https://github.com/SpongePowered/SpongeForge/issues/2399 so it might be better to handle that in a different PR.

**EDIT:** ready to review and merge

**Review by RedNesto — APPROVED** [review]
Seems good to me.
I used the updated buildscripts and they were working fine.

**Pull request merged**